setup.js

```diff
@@ -0,0 +1,108 @@
+import { Nation } from "./constants";
+
+const error = (want) => (x) => {
+  throw new Error(`got=${x.value}, want=${want}`);
+};
+
+export default ({ players }) => {
```

Comment:
It's totally fine if this is more effort than it's worth but perhaps our `imperial.test.js` file is really testing `setup.js`? Do you think it would take an inordinate amount of effort to re-purpose `imperial.test.js` to test the setup? And if we do that would we still need to test that `Imperial`'s constructor works?

Ultimately, I'm sure a done thing is a good thing but I'm just curious about the various merits and demerits to each approach.

Reply:
I'll rename `imperial.test.js` to `setup.test.js` and slightly modify the scope of those tests. I think we can test the integration between `setup` and `imperial` when we replace the setup-like actions in the Schnelleinsteig. Sound good?

Reply:
Yep, sounds great to me!